src/Microsoft.Dnx.Tooling/Restore/RestoreProjectsCollector.cs

```diff
@@ -0,0 +1,54 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+
+namespace Microsoft.Dnx.Tooling
+{
+    internal class RestoreProjectsCollector
+    {
+        public static bool Find(string restoreDirectory, out IEnumerable<string> projectJsonFiles)
+        {
+            if (string.Equals(Runtime.Project.ProjectFileName,
+                              Path.GetFileName(restoreDirectory),
+                              StringComparison.OrdinalIgnoreCase) && File.Exists(restoreDirectory))
```

Comment:
Now including globbing, we compare file in case-insensitive way across platforms. It's not ideal but consistent.